### PR TITLE
Fix occasional ci error at InventoryTaskTest

### PR DESCRIPTION
**Pull request by azexcy**
link ci error: https://github.com/apache/shardingsphere/actions/runs/3709680170

Changes proposed in this pull request:
  - Fix occasional ci error at InventoryTaskTest

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

**Review by sandynz — APPROVED** [review]

File: test/it/pipeline/src/test/java/org/apache/shardingsphere/test/it/data/pipeline/core/task/InventoryTaskTest.java [hunk 1]
```diff
@@ -77,7 +76,7 @@ public void assertStartWithGetEstimatedRowsFailure() throws ExecutionException,
         PipelineTableMetaDataLoader metaDataLoader = new StandardPipelineTableMetaDataLoader(dataSource);
         try (
                 InventoryTask inventoryTask = new InventoryTask(inventoryDumperConfig, taskConfig.getImporterConfig(),
-                        PipelineContextUtil.getPipelineChannelCreator(), new DataSourceImporterConnector(DATA_SOURCE_MANAGER), dataSource,
```
Comment (azexcy) [inline test/it/pipeline/src/test/java/org/apache/shardingsphere/test/it/data/pipeline/core/task/InventoryTaskTest.java:80]: TODO, In concurrent scenarios, importer may not stop causing ci timeout, I will follow up on this

**Comment by azexcy** [comment]
TODO, In concurrent scenarios, importer may not stop causing ci timeout, I will follow up on this
